mindsdb/libs/controllers/functional.py

```diff
@@ -0,0 +1,178 @@
+import os
+import pickle
+import shutil
+import zipfile
+
+from mindsdb.config import CONFIG
+
+
+def export_storage(mindsdb_storage_dir='mindsdb_storage'):
+    """
+    If you want to export this mindsdb's instance storage to a file
+
+    :param mindsdb_storage_dir: this is the full_path where you want to store a mind to, it will be a zip file
+    :return: bool (True/False) True if mind was exported successfully
+    """
+    shutil.make_archive(base_name=mindsdb_storage_dir, format='zip',
+                        root_dir=CONFIG.MINDSDB_STORAGE_PATH)
+    print(f'Exported mindsdb storage to {mindsdb_storage_dir}.zip')
+
+
+def export_predictor(model_name):
+    """Exports a Predictor to a zip file in the CONFIG.MINDSDB_STORAGE_PATH directory.
+
+    :param model: a Predictor
+    :param model_name: this is the name of the model you wish to export (defaults to the name of the passed Predictor)
+    """
+    storage_file = model_name + '.zip'
+    with zipfile.ZipFile(storage_file, 'w') as zip_fp:
+        for file_name in [model_name + '_heavy_model_metadata.pickle',
+                          model_name + '_light_model_metadata.pickle',
+                          model_name + '_lightwood_data']:
+            full_path = os.path.join(CONFIG.MINDSDB_STORAGE_PATH, file_name)
+            zip_fp.write(full_path, os.path.basename(full_path))
+
+        # If the backend is ludwig, save the ludwig files
+        try:
+            ludwig_model_path = os.path.join(CONFIG.MINDSDB_STORAGE_PATH,
+                                             model_name + '_ludwig_data')
+            for root, dirs, files in os.walk(ludwig_model_path):
+                for file in files:
+                    full_path = os.path.join(root, file)
+                    zip_fp.write(full_path,
+                                 full_path[len(CONFIG.MINDSDB_STORAGE_PATH):])
+        except Exception:
+            pass
+
+    print(f'Exported model to {storage_file}')
+
+
+def rename_model(old_model_name, new_model_name):
+    """
+    If you want to rename an exported model.
+
+    :param old_model_name: this is the name of the model you wish to rename
+    :param new_model_name: this is the new name of the model
+    :return: bool (True/False) True if predictor was renamed successfully
+    """
+
+    if old_model_name == new_model_name:
```

Comment:
I think we should raise an exception when you attempt to rename a model that doesn't exist

Reply:
it can be done after merge I guess, everything else looks good to me.

Reply:
Done